### PR TITLE
Make interface constants final

### LINKED ISSUE #61: make constant final
right now the constants arent final.
in order to prevent change of them or mistakes, make them final!

### PR DESCRIPTION
**Pull request by eyal1889**
now all the lua interface's constants are real constants!
no more bugs caused by changing them! :)

### CODE REVIEW

File: intf/cnsr_intf.lua [hunk 1]
```diff
@@ -201,7 +213,8 @@ function skip(skip_start, skip_end)
 	prev_time = current_time
 	local skip_length = skip_end - skip_start
 	if forward then
-		current_time = math.min(current_time + skip_length + SKIP_SAFETY, vlc.input.item():duration() * MS_IN_SEC) --think if we want to!
+		skip_length=skip_end-math.max(skip_start,current_time)
```
Comment (ophirhan) [inline intf/cnsr_intf.lua]: ?

Comment (eyal1889) [reply intf/cnsr_intf.lua]: ye, well i thought i fixed it but it does the same as the original idea. so pls dont take those lines

File: modules/constants.lua [hunk 2]
```diff
@@ -0,0 +1,9 @@
+CONSTANTS={}
```
Comment (ishayhil) [inline modules/constants.lua]: why not put all constants here? Currently this file looks redundant. Also, no need for get_constants function. Just wrap return the wrapped protected table.

**Review by ishayhil — DISMISSED** [review]

File: modules/constants.lua (hunk 2 above)
Comment (eyal1889) [reply modules/constants.lua]: me and ophir talked and we relized that we need those constants only for the interface, so this file is not needed and should be deleted.

**Review by ophirhan — APPROVED** [review]

**Pull request merged**